### DISCUSSION TITLE
Extract time series in a csv file

### DISCUSSION THREAD

**Discussion by costantino78** [insarlab/MintPy#814] [Q&A] [answered]
Hello everybody.
I want to extract time series data into a csv file. I used `read_timeseries_lalo()` to extract data of one point, how can I extract all points of my time series? I would like a csv like this:

```
POINT_1_LAT | POINT_1_LON | DISPLACEMENT_DATE_1 | DISPLACEMENT_DATE_2 | DISPLACEMENT_DATE_3 | ...
POINT_2_LAT | POINT_2_LON | DISPLACEMENT_DATE_1 | DISPLACEMENT_DATE_2 | DISPLACEMENT_DATE_3 | ...
POINT_3_LAT | POINT_3_LON | DISPLACEMENT_DATE_1 | DISPLACEMENT_DATE_2 | DISPLACEMENT_DATE_3 | ...
...
```

I saw this notebook https://nbviewer.org/github/insarlab/MintPy-tutorial/blob/main/io/save_text.ipynb but I don't understand how it works.

Regards
Costantino

**Comment by hardreddata** [comment, marked as answer]
If you are using QGIS (and so `save_qgis.py`) you can export the MintPy shapefile to .csv inside QGIS making sure to set the `GEOMETRY=AS_XY` option so you get the latitude and longitude.

**Comment by costantino78** [comment]
Thanks for the reply, it works.
